integration_tests/transition_multicopter_fixedwing.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include "integration_test_helper.h"
+#include "dronecore.h"
+
+using namespace dronecore;
+
+
+//static void connect(DroneCore);
+static void takeoff(Device&);
+static void takeoff_and_transition_to_fixedwing();
+static void land_and_disarm(Device&);
+
+
+TEST_F(SitlTest, ActionSimpleTransition)
+{
+    takeoff_and_transition_to_fixedwing();
+}
+
+void takeoff_and_transition_to_fixedwing()
+{
+	// Init & connect
+    DroneCore dc;
+
+	DroneCore::ConnectionResult ret = dc.add_udp_connection();
+    ASSERT_EQ(ret, DroneCore::ConnectionResult::SUCCESS);
+
+    // Wait for device to connect via heartbeat.
+    std::this_thread::sleep_for(std::chrono::seconds(2));
+    ASSERT_TRUE(dc.is_connected());
+
+	Device &device = dc.device();
+
+	// Takeoff
```

Comment:
I would get rid of comments that are identical to a function name. They don't add any information. I suggest only to add comments if you need to explain something, add additional info for the reader, e.g. `We need to takeoff first, otherwise we can't actually transition` 😄